test/python/docker/compat/test_containers.py

```diff
@@ -302,3 +309,33 @@ def wait_and_start():
         finally:
             ctr.stop()
             ctr.remove(force=True)
+
+    def test_container_inspect_compatibility(self):
```

Comment:
This is not really an API test as you check `podman inspect` output against the docker schema which is not a bad idea at all but the API test is checking the output/behavior of our API but I am also not sure if there is a better place right now.

Do you need to do this in python? I think if we want to validate this it would be better in a go unit test.

Reply:
Apparently there is a module for go:  https://github.com/go-openapi/validate. 
But it's not compatible with openapi v3, so if docker change the format from v2 to v3 it wont work anymore.
If you think it's mandatory to do this test in go can you tell me where I should add it ?